src/features/events/components/EventPopper/SingleEventPopper.tsx

```diff
@@ -0,0 +1,333 @@
+import { makeStyles } from '@mui/styles';
+import NextLink from 'next/link';
+import { useRouter } from 'next/router';
+import {
+  AccessTime,
+  ArrowForward,
+  EmojiPeople,
+  FaceOutlined,
+  MailOutline,
+  People,
+  PlaceOutlined,
+} from '@mui/icons-material';
+import {
+  Box,
+  Button,
+  Checkbox,
+  ClickAwayListener,
+  Link,
+  Paper,
+  Popover,
+  Typography,
+} from '@mui/material';
+import { FC, useContext } from 'react';
+
+import LocationName from '../LocationName';
+import messageIds from 'features/events/l10n/messageIds';
+import Quota from './Quota';
+import StatusDot from './StatusDot';
+import { useMessages } from 'core/i18n';
+import useModel from 'core/useModel';
+import { ZetkinEvent } from 'utils/types/zetkin';
+import { ZUIConfirmDialogContext } from 'zui/ZUIConfirmDialogProvider';
+import ZUIEllipsisMenu from 'zui/ZUIEllipsisMenu';
+import ZUIIconLabel from 'zui/ZUIIconLabel';
+import ZUIPerson from 'zui/ZUIPerson';
+import ZUIPersonHoverCard from 'zui/ZUIPersonHoverCard';
+import ZUITimeSpan from 'zui/ZUITimeSpan';
+import EventDataModel, {
+  EventState,
+} from 'features/events/models/EventDataModel';
+
+const useStyles = makeStyles(() => ({
+  description: {
+    '-webkit-box-orient': 'vertical',
+    '-webkit-line-clamp': 3,
+    display: '-webkit-box',
+    maxHeight: '100%',
+    overflow: 'hidden',
+    whiteSpace: 'normal',
+    width: '100%',
+  },
+}));
+
+export interface SingleEventPopperProps {
+  anchorPosition: { left: number; top: number } | undefined;
+  event: ZetkinEvent;
+  onClickAway: () => void;
+  open: boolean;
+}
+
+const SingleEventPopper: FC<SingleEventPopperProps> = ({
```

Comment:
I notice that this is very similar to `MultiEventPopper/SingleEvent.tsx`. Would it be possible to reuse some part of the content?

When I remove the popover, paper and `ClickAwayListener` wrappers in this component, the diff becomes very small, making me think that it's mostly reusable:

```diff
2a3
> import { useMessages } from 'core/i18n';
13,22c14
< import {
<   Box,
<   Button,
<   Checkbox,
<   ClickAwayListener,
<   Link,
<   Paper,
<   Popover,
<   Typography,
< } from '@mui/material';
---
> import { Box, Button, Checkbox, Link, Typography } from '@mui/material';
25c17
< import LocationName from '../LocationName';
---
> import LocationName from '../../LocationName';
27,29c19,20
< import Quota from './Quota';
< import StatusDot from './StatusDot';
< import { useMessages } from 'core/i18n';
---
> import Quota from '../Quota';
> import StatusDot from '../StatusDot';
54,55c45
< export interface SingleEventPopperProps {
<   anchorPosition: { left: number; top: number } | undefined;
---
> interface SingleEventProps {
58d47
<   open: boolean;
61,66c50
< const SingleEventPopper: FC<SingleEventPopperProps> = ({
<   anchorPosition,
<   event,
<   onClickAway,
<   open,
< }) => {
---
> const SingleEvent: FC<SingleEventProps> = ({ event, onClickAway }) => {
69d52
<   const classes = useStyles();
70a54,55
>   const classes = useStyles();
> 
126c111
<     <Box display="flex" flexDirection="column">
---
>     <>
295c280
<     </Box>
---
>     </>
299c284
< export default SingleEventPopper;
---
> export default SingleEvent;

```

Reply:
Yep, this has been at the back of my mind and I've just ignored it and forgotten to do something about it heh